### DISCUSSION TITLE
youtube sign in to confirm your age

### DISCUSSION THREAD

**Discussion by skijbal** [RandomNinjaAtk/docker-lidarr-extended#127] [Q&A] [answered]
I have noticed in the logs that I'm seeing the following error:
**ERROR: [youtube] Wh8DT09QCHI: Sign in to confirm your age. This video may be inappropriate for some users.**

Is there a way to address this and correct it or something we need to live with.   

Thanks

**Comment by RandomNinjaAtk** [comment, marked as answer]
You need to create a `cookies.txt` file that will be used by yt-dlp for authentication and place it in the `/config` folder.

For more information: https://github.com/yt-dlp/yt-dlp/wiki/FAQ#how-do-i-pass-cookies-to-yt-dlp

**Reply by skijbal** [reply]
Thanks for the help
